odoo_2_odoo_data_transfer/wizards/odoo_data_transfer_wizard_views.xml

```diff
@@ -39,19 +39,20 @@
                         </group>
                     </group>
                     <group>
-                        <field name="template_id" />
+                        <field name="state" invisible="True" />
+                        <field name="template_id" invisible="state =='validated'" />
                         <field
                             name="remote_source_model_name"
-                            invisible="not template_id"
+                            invisible="state !='validated'"
                         />
                         <field
                             name="local_target_model_id"
-                            invisible="not template_id"
+                            invisible="state !='validated'"
```

Comment:
It should be invisible if state == 'validated (although I'd keet it always visible)

Reply:
The state becomes validated once you select the template and you click the validate button.

If you change this field value after selecting a template, the value will be overriden before, so it does only make sense to be seen then the state is validated

Reply:
At the end i have done what you requested